docs/DeveloperGuide.md

```diff
@@ -1,38 +1,662 @@
+
+###### [Back to table of contents](#table-of-contents)
+
+### View History
+
+###### [Back to table of contents](#table-of-contents)
+
+### View Latest
+
+###### [Back to table of contents](#table-of-contents)
+
+### Delete Item
+
+###### [Back to table of contents](#table-of-contents)
+
+### Storage of Data
+
+###### [Back to table of contents](#table-of-contents)
+
+## Appendix: Requirements
+
+* [Product Scope](#product-scope)
+    * [Target User Profile](#target-user-profile)
+    * [Value Proposition](#value-proposition)
+* [User Stores](#user-stories)
+* [Non-Functional Requirements](#non-functional-requirements)
+* [Glossary](#glossary)
+
+### Product scope
+
+#### Target user profile
+
+Outpatients who need to monitor their health activity and health parameters.
+
+###### [Back to table of contents](#table-of-contents)
+
+#### Value proposition
+
+PulsePilot is a health monitoring application designed to bridge the gap between medical professionals and patients during outpatient recovery.
+
+PulsePilot offers outpatients the capability to input and track a range of health activities, encompassing both aerobic and anaerobic exercises, alongside crucial health parameters such as BMI and menstrual cycles.
+
+Simultaneously, PulsePilot facilitates access to this vital data for various healthcare professionals, ensuring comprehensive and seamless support in guiding outpatient recovery processes.
+
+###### [Back to table of contents](#table-of-contents)
+
+### User Stories
+
+| Version | As a ...              | So that I can ...                                     | I want to ...                         |
+|---------|-----------------------|-------------------------------------------------------|---------------------------------------|
+| 1.0     | gym enthusiast        | track my gym sessions                                 | enter my gym stats                    |
+| 1.0     | runner                | see my relative speed for each run                    | see my running pace                   |
+| 1.0     | runner                | track my running progress over time                   | log my runs                           |
+| 1.0     | health conscious user | track change in my weight over time                   | calculate my BMI                      |
+| 1.0     | female user           | monitor any deviations from my normal menstrual cycle | track my menstrual cycle              |
+| 2.0     | runner                | quickly view my most recent run details               | see my latest run                     |
+| 2.0     | gym enthusiast        | quickly view my most recent gym session               | see my latest gym session             | 
+| 2.0     | gym enthusiast        | accurately track my progress and strength gains       | enter varying weights for sets        | 
+| 2.0     | female user           | plan ahead and better manage my health                | p#D85D43ict my next period start date | 
```

Comment:
seems like a formatting/typo error on the female user row?


![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/61624545/86e14977-377a-47a5-a579-cec34225fc51)